### PR TITLE
5 Min SketchFab to A-Frame Guide (draft)

### PR DESCRIPTION
**Pull request by 321C4**
Hey guys,
This article is done, except paths to image files need tweaking..

### CODE REVIEW

File: creating/export-blender-to-aframe/index.md [hunk 1]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+
+
+# PART 1: GETTING THE “INGREDIENTS”
+---
+
+## Get Blender
+
+![Image of Blender Home Page](https://github.com/WebVRRocks/webvrrocks/tree/master/creating/export-blender-to-aframe/img/BlenderSite.png)
+
+Go to https://www.blender.org/ and click on the big blue Download Blender button. You’ll come to the downloads page, where you can choose which installer to download, depending on your operating system. Mac, Windows and GNU/Linux are supported. Choose a mirror to download from (probably one closeby). Once you’ve downloaded the installer, open it, and click through the steps of the installation wizard. On Windows, Blender should end up in C:\Program Files\Blender Foundation\Blender.
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: I feel like this would read better as an ordered list:

0. Go to the [Downloads page on Blender.org](https://www.blender.org/download/).
0. Download the version of Blender for your platform (Windows, macOS, Linux).
0. Install the Blender application.
    * **Windows:** Open the `.msi` file and follow the install wizard ([read full instructions](https://www.blender.org/manual/getting_started/installing/windows.html)).
    * **macOS:** Double-click on the `.zip` file to uncompress it, drag the `Blender` and `BlenderPlayer` applications to your `Applications` folder ([read full instructions](https://www.blender.org/manual/getting_started/installing/macos.html)).
    * **Linux:** Uncompress the `.zip` file to your desired location, such as `~/software/` or `/usr/local/` ([read full instructions](https://www.blender.org/manual/getting_started/installing/linux.html)).
0. Launch Blender.
    * **Windows:** `C:\Program Files\Blender Foundation\Blender`
    * **macOS:** `~/Applications/Blender.app`
    * **Linux:** `/usr/local/Blender.app`

<hr>

here's the Markdown source:

```md
0. Go to the [Downloads page on Blender.org](https://www.blender.org/download/).
0. Download the version of Blender for your platform (Windows, macOS, Linux).
0. Install the Blender application.
    * **Windows:** Open the `.msi` file and follow the install wizard ([read full instructions](https://www.blender.org/manual/getting_started/installing/windows.html)).
    * **macOS:** Double-click on the `.zip` file to uncompress it, drag the `Blender` and `BlenderPlayer` applications to your `Applications` folder ([read full instructions](https://www.blender.org/manual/getting_started/installing/macos.html)).
    * **Linux:** Uncompress the `.zip` file to your desired location, such as `~/software/` or `/usr/local/` ([read full instructions](https://www.blender.org/manual/getting_started/installing/linux.html)).
0. Launch Blender.
    * **Windows:** `C:\Program Files\Blender Foundation\Blender`
    * **macOS:** `~/Applications/Blender.app`
    * **Linux:** `/usr/local/Blender.app`
```

File: creating/export-blender-to-aframe/index.md [hunk 2]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+# PART 1: GETTING THE “INGREDIENTS”
+---
+
+## Get Blender
+
+![Image of Blender Home Page](https://github.com/WebVRRocks/webvrrocks/tree/master/creating/export-blender-to-aframe/img/BlenderSite.png)
+
+Go to https://www.blender.org/ and click on the big blue Download Blender button. You’ll come to the downloads page, where you can choose which installer to download, depending on your operating system. Mac, Windows and GNU/Linux are supported. Choose a mirror to download from (probably one closeby). Once you’ve downloaded the installer, open it, and click through the steps of the installation wizard. On Windows, Blender should end up in C:\Program Files\Blender Foundation\Blender. 
+
+## Make a directory with an index.html file in it
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: backtick `index.html`

File: creating/export-blender-to-aframe/index.md [hunk 3]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+Here is some handy boilerplate code to use. make a new  file in your favorite text editor, and copy this into it. I like Sublime Text quite a bit, but any text editor is fine. Name it index.html so that the server can find it. We’ll come back to the code, later.
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: uppercase "make" and remove space after "new"

Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: remove comma before `later`

File: creating/export-blender-to-aframe/index.md [hunk 4]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+
+'''
+<!--.....................................................
+Attribution for 3D model used in this scene: Oculus Rift CV1 by Ti is licensed under CC Attribution-NonCommercial
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: please link to Sketchfab model page, user's Sketchfab profile page, and Creative Commons license URL (with the version number)

File: creating/export-blender-to-aframe/index.md [hunk 5]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+.........................................................-->
+
+<!DOCTYPE html>
+   <head>
+    <meta charset="utf-8">
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: can you add back the `<meta name="viewport" …>`?

File: creating/export-blender-to-aframe/index.md [hunk 6]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+    <title>Exporting Blender Models from Sketchfab to A-Frame – WebVR Rocks</title>
+    <script src="assets/js/aframe.min.js"></script>
+    <link rel="icon" sizes="192x192" href="https://aframe.io/images/aframe-logo-192.png">
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: do you need this?

File: creating/export-blender-to-aframe/index.md [hunk 7]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+    <a-asset-item id="Oculus-CV1-obj" src="cv1_FF.obj"></a-asset-item>
+    <a-asset-item id="Oculus-CV1-mtl" src="cv1_FF.mtl"></a-asset-item>
+    <a-asset-item id="Oculus-CV1-collada" src="cv1_FF.dae"></a-asset-item>
+    </a-assets>
+
+<!--Sky color can be adjusted-->
+    <a-sky color="#5CAAEB"></a-sky>
+
+<!--Camera has WASD controls by default. In desktop (non VR) mode, click and drag to look around.-->
+    <a-camera position="0, 1.8, 5"></a-camera>
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: remove commas

File: creating/export-blender-to-aframe/index.md [hunk 8]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+## Get Blender
+
+![Image of Blender Home Page](https://github.com/WebVRRocks/webvrrocks/tree/master/creating/export-blender-to-aframe/img/BlenderSite.png)
+
+Go to https://www.blender.org/ and click on the big blue Download Blender button. You’ll come to the downloads page, where you can choose which installer to download, depending on your operating system. Mac, Windows and GNU/Linux are supported. Choose a mirror to download from (probably one closeby). Once you’ve downloaded the installer, open it, and click through the steps of the installation wizard. On Windows, Blender should end up in C:\Program Files\Blender Foundation\Blender. 
+
+## Make a directory with an index.html file in it
+Here is some handy boilerplate code to use. make a new  file in your favorite text editor, and copy this into it. I like Sublime Text quite a bit, but any text editor is fine. Name it index.html so that the server can find it. We’ll come back to the code, later.
+
+'''
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: this look like this:

    ```html
    …
    ```

File: creating/export-blender-to-aframe/index.md [hunk 9]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+    <link rel="icon" sizes="192x192" href="https://aframe.io/images/aframe-logo-192.png">
+   </head>
+  <body>
+<!--All of the entities in your scene must be between the <a-scene> tags!-->
+    <a-scene>
+<!--......................................
+It's a good idea to pre-load and cache your assets (stuff in the scene) by using an Asset System. This is an example of an Asset System. Everything in <a-assets> can be used as many times as you want in the scene! You can also use entities inline, without an Asset System: quicker to write, but slower to render. <asset-item> can include geometry, models, and materials
+..........................................-->
+    <a-assets>
+    <a-asset-item id="Oculus-CV1-obj" src="cv1_FF.obj"></a-asset-item>
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: indent, etc.

File: creating/export-blender-to-aframe/index.md [hunk 10]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+<!--Here are the .obj and .dae models.-->
+      <a-entity obj-model="obj: #Oculus-CV1-obj; mtl: #Oculus-CV1-mtl" position="-5 0 -5"></a-entity>
+      <a-entity collada-model="#Oculus-CV1-collada" position="5 0 -5"></a-entity>
+
+    </a-scene>
+  </body>
+</html>
+'''
+
+## Get npm (if you don’t have it already)
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: you can remove the parenthetical statement

File: creating/export-blender-to-aframe/index.md [hunk 11]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+npm provides an easy way to install packages from the command line, using the Node.js Javascript runtime.
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: can you link to both? and `Javascript` -> `JavaScript`

File: creating/export-blender-to-aframe/index.md [hunk 12]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+
+## Get npm (if you don’t have it already)
+npm provides an easy way to install packages from the command line, using the Node.js Javascript runtime. 
+If you are on Windows or Mac, installing the latest version of Node will also install npm on your machine.
+
+After installing Node, check that it’s installed properly and check the version of Node and npm. Open command prompt and type:
+
+'''
+node -v 
+nmp -v
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: typo: should be `npm`

File: creating/export-blender-to-aframe/index.md [hunk 13]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+'''
+
+Each of these commands should print a version number.
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: what's the purpose of telling folks how to check the Node and npm version? is it to see if it's installed? IMO, I think we can leave out that part.

File: creating/export-blender-to-aframe/index.md [hunk 14]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+
+'''
+node -v 
+nmp -v
+'''
+
+Each of these commands should print a version number. 
+
+## Get http-server
+You can use any local dev server you are comfortable with. Here, I chose http-server, because it’s simple to install with npm, and easy to use.
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: can you link to it?

File: creating/export-blender-to-aframe/index.md [hunk 15]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+To install the server globally, simply enter this on the command line:
+npm install http-server -g
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: this needs to be like this with a separate line in between:

    ```sh
    npm install http-server -g
    ```

File: creating/export-blender-to-aframe/index.md [hunk 16]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+
+To start the server and open a browser showing your web page:
+http-server [full/path/to/index.html] -o
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: same here

Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: you should always invoke `http-server` with `http-server -c-1` to disable caching for local development

File: creating/export-blender-to-aframe/index.md [hunk 17]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+
+There are other handy options for the server, specified on https://www.npmjs.com/package/http-server
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: I would use Markdown syntax to link to it; no need to print out the whole URL. and end with a period

File: creating/export-blender-to-aframe/index.md [hunk 18]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+
+Make a directory to keep your blend file and exports
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: blend -> `.blend` (with backticks)

Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: add a period

File: creating/export-blender-to-aframe/index.md [hunk 19]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+We’ll come back to this directory later.
+
+
+# PART 2: GET AND CONVERT THE 3D MODEL
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: title-case here too

File: creating/export-blender-to-aframe/index.md [hunk 20]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+---
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: remove these lines

File: creating/export-blender-to-aframe/index.md [hunk 21]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+
+## Choose a blend file on SketchFab
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: lowercase the "F"

File: creating/export-blender-to-aframe/index.md [hunk 22]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+For this guide, I’m using this CV1 model: 
+https://sketchfab.com/models/591ff55a0f7b401cbb14fe454175b8b2
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: use Markdown link syntax

File: creating/export-blender-to-aframe/index.md [hunk 23]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+Note that the model
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: note what?

File: creating/export-blender-to-aframe/index.md [hunk 24]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+# PART 2: GET AND CONVERT THE 3D MODEL
+---
+
+## Choose a blend file on SketchFab
+For this guide, I’m using this CV1 model: 
+https://sketchfab.com/models/591ff55a0f7b401cbb14fe454175b8b2
+Note that the model 
+
+## Download the blend file
+It will be a zip file.
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: I feel like these should be itemized in an ordered list, à la the Blender instructions above

File: creating/export-blender-to-aframe/index.md [hunk 25]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+
+## Unzip/extract the blend file
+Right click on the .zip file and select Extract All. Save the extracted contents to a directory you can find easily.
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: `select Extract All.` -> `press the "Extract All" button.`

Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: you mentioned above, but I would be explicit again here that this is specific to Windows

File: creating/export-blender-to-aframe/index.md [hunk 26]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+
+![Extract Zip Contents](https://github.com/WebVRRocks/webvrrocks/tree/master/creating/export-blender-to-aframe/img/ExtractAll.png)
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: here too change the image URL: `/creating/export-blender-to-aframe/img/extract-all.png`

File: creating/export-blender-to-aframe/index.md [hunk 27]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+
+## Open the blend file with Blender
+Take a quick look at the Blender user interface. It has a high learning curve, but can be a powerful tool for creating 3D graphics. For now, we’ll just use it as an exporter for the CV1 model.
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: `Take a quick look at the Blender user interface. It has a` -> `Blender has a`

File: creating/export-blender-to-aframe/index.md [hunk 28]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+
+## Export the model: File → Export → Wavefront (.obj)
+Name and save the .obj file (saved by default in same directory with the blend file)
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: end with punctuation

File: creating/export-blender-to-aframe/index.md [hunk 29]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+
+![Export .obj from Blender](https://github.com/WebVRRocks/webvrrocks/tree/master/creating/export-blender-to-aframe/img/ExportOBJ.png)
+
+## Copy the .obj and .mtl files into the directory with your index.html file
+Go back to the blend file in Blender and export as Collada (.dae)
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: end with punctuation, and use an ordered list

File: creating/export-blender-to-aframe/index.md [hunk 30]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+In the directory with your blend file, you should now have a .dae file, plus a .obj file and a .mtl file that goes with it. Note that there are a number of .jpg image files that came with the model. These are textures for the .obj: they are important! Copy-paste the .dae, .obj .mtl and all of the .jpg files into the directory with your index.html file.
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: backtick all the file extensions

Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: a .obj -> an `.obj` (with backticks)

Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: `: they are important` -> `; they are important`

Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: `Copy-paste` -> `Copy-and-paste`

Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: `.obj .mtl and all` -> `.obj, .mtl, and all`

File: creating/export-blender-to-aframe/index.md [hunk 31]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+Name and save the .obj file (saved by default in same directory with the blend file)
+
+![Export .obj from Blender](https://github.com/WebVRRocks/webvrrocks/tree/master/creating/export-blender-to-aframe/img/ExportOBJ.png)
+
+## Copy the .obj and .mtl files into the directory with your index.html file
+Go back to the blend file in Blender and export as Collada (.dae)
+In the directory with your blend file, you should now have a .dae file, plus a .obj file and a .mtl file that goes with it. Note that there are a number of .jpg image files that came with the model. These are textures for the .obj: they are important! Copy-paste the .dae, .obj .mtl and all of the .jpg files into the directory with your index.html file.  
+
+
+# PART 3: WEB PAGE AND LOCAL SERVER
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: same

File: creating/export-blender-to-aframe/index.md [hunk 32]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+---
+
+## Open your index.html file in your favorite text editor
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: you could rephrase as "Open `index.html` in a text editor"

File: creating/export-blender-to-aframe/index.md [hunk 33]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+In the boilerplate, note that I’ve provided attribution to the model’s creator in a comment in lines 1-3. If you are new to A-Frame, there are some helpful comments in the code, as well.
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: `note that` should be `notice that`

Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: `in lines` -> `on lines`

File: creating/export-blender-to-aframe/index.md [hunk 34]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+Because the 3D model files are in the same directory with your index.html, the file paths to them simply consist of their names. Go to line 26, which is within <a-assets>. This is  where we’ll initially load the assets for the scene. In src=”cv1_FF.obj” change cv1_FF.obj to match the name you gave your .obj file. Do the same for the .mtl and the .dae asset items in lines 27 and 28.
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: is this first sentence necessary?

Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: remove the extra space after `is`

Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: backtick appropriate things here

Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: change the ” and ” to "

Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: "the name you gave your .obj file" -> "the `.obj` filename"

Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: `in lines` -> `on lines`

File: creating/export-blender-to-aframe/index.md [hunk 35]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+Note that each model is given an ID, which is used to instantiate them into the scene as entities in lines 39 and 40. Using an asset system like this has the advantage of pre-loading your models, and you can then instantiate them as many times as you want without sacrificing as much performance.
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: `Notice that`

Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: `in lines` -> `on lines`

Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: `pre-loading` -> preloading

Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: these two sentences are almost the same; can you merge them?

File: creating/export-blender-to-aframe/index.md [hunk 36]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+
+## Start up your local http server 
+Open the command prompt. Type:
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: can you recommend using Git Bash (with a link) as a preferred command-line interface on Windows?

File: creating/export-blender-to-aframe/index.md [hunk 37]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+
+'''
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: use ```sh

File: creating/export-blender-to-aframe/index.md [hunk 38]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+http-server [full\path\to\directory containing index.html] -o
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: can you show your example path (e.g., C:\Users\myname\projects\export-blender-to-aframe\index.html`)?

File: creating/export-blender-to-aframe/index.md [hunk 39]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+'''
+
+The -o option will open a default browser with your index.html webpage. The default port it uses is 8080; there’s a option to change this, if you want. https://www.npmjs.com/package/http-server
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: backtick `-o`, etc.

Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: isn't this already covered above?

File: creating/export-blender-to-aframe/index.md [hunk 40]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+
+## Can you see the models in the webpage?
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: possibly rename to `Testing your scene`

File: creating/export-blender-to-aframe/index.md [hunk 41]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+The Collada .dae will be on your left, and the Wavefront .obj on your left. Click-and-drag to rotate the camera and look around. You can also move with the traditional W-A-S-D keys. If you can’t see the models, check the file paths in <a-assets> and also the IDs in the obj-model and collada-model entities.
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: the terminology `WASD controls` is used above, as opposed to `W-A-S-D keys` here; we should be consistent

Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: `COLLADA` should always be all uppercased everywhere

Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: IMO, it's a bit odd that this is the firs time `Wavefront .obj` is mentioned (as opposed to just `.obj`)

Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: can you rephrase the first sentence to just say that both will be on the user's left?

Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: `<a-assets>` needs to be backticked or the Markdown parser will try to render it as HTML

Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: backtick the `obj-model`, `collada-model`, etc.

Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: entities -> A-Frame components (HTML attributes) on the entities (i.e., the `<a-entity>` HTML elements)

File: creating/export-blender-to-aframe/index.md [hunk 42]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+
+VR viewing of your locally served site can be done with either a phone + the right ip address + cardboard, or a vr-enabled browser. For quick viewing on Cardboard: In the command prompt with the http-server running, you should see two urls that each contain an ip address and a port number. Enter the second one into the address bar of your phone browser, and hit ‘go’.
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: can you rephrase "VR viewing"

Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: can you show how to use to load this on a phone? for this reason, I would recommend [`browser-sync`](https://www.browsersync.io/docs/command-line) instead of `http-server` to do easy local tunneling and show local IP addresses for easy development.

Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: `IP address` should always be capitalised

Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: I would remove "hit go" and just say "load the URL from a mobile browser"

File: creating/export-blender-to-aframe/index.md [hunk 43]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
...
+
```
Comment (cvan) [inline creating/export-blender-to-aframe/index.md]: remove these extra newlines (but keep one newline at the end of the file)

File: creating/export-blender-to-aframe/demo/index.html [hunk 44]
```diff
@@ -1,34 +1,36 @@
...
-<html>
-  <head>
+<!--.....................................................
+Attribution for 3D model used in this scene: Oculus Rift CV1 by Ti is licensed under CC Attribution-NonCommercial
+.........................................................-->
+
+<!DOCTYPE html>
+   <head>
     <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (321C4) [inline creating/export-blender-to-aframe/demo/index.html]: done

File: creating/export-blender-to-aframe/index.md [hunk 45]
```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
 ## Finding a model on Sketchfab
 
 ## View the demo
+
+In this guide, we’ll cook up an A-Frame scene containing a model from SketchFab. I chose a model of a Rift CV1 by Ti Kawamoto. It’s a nice model created with Blender, that goes with the theme of WebVR. Plus, the thought of recursive “VR within VR” is amusing..
+For this lesson, I’m using Windows because that is what most VR devs use, but do please reach out to me if you have any questions about doing this in another OS! If you are comfortable setting up a site on a local dev server, feel free to ignore the parts about setting up http-server.
```
Comment (321C4) [inline creating/export-blender-to-aframe/index.md]: yep, twitter.

I'm probably going to re-do this with browser-sync actually, since http-server doesn't seem to work to view locally on a phone! Yes, I will link to the repo; there's some useful stuff in the org, though: https://github.com/BrowserSync

File: creating/export-blender-to-aframe/index.md (hunk 36 above)
Comment (321C4) [inline creating/export-blender-to-aframe/index.md]: Shall I explain Git Bash here, as well? Or, just give a recommendation with link to https://git-scm.com/, and assume people will figure it out?

File: creating/export-blender-to-aframe/index.md (hunk 41 above)
Comment (321C4) [inline creating/export-blender-to-aframe/index.md]: I'm confused by this: "entities -> A-Frame components (HTML attributes) on the entities (i.e., the <a-entity> HTML elements)"

Do you mean that I should refer to the entities that contain the models as "A-Frame components"?

File: creating/export-blender-to-aframe/index.md (hunk 43 above)
Comment (321C4) [inline creating/export-blender-to-aframe/index.md]: my demo breaks when I use that minimized version of a-frame in the js directory.. going back to using <script src="https://aframe.io/releases/0.3.2/aframe.min.js"></script>

**Comment by 321C4** [comment]
@cvan I made the requested edits and switched to using Browsersync. Please take a look.

**Comment by cvan** [comment]
thanks, looks great. merging so we can adjust and polish on `master` 👍
